lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js

```diff
@@ -270,51 +270,3 @@ test('it does not attach json to submission when env var not present', async t =
   submitterClientSpy.resetHistory()
   t.end()
 })
-
-test('TEAM PDF: it attaches PDF contents to email submission attachments', async t => {
-  getUserDataPropertyStub.resetHistory()
-
-  const pageSummaryController = proxyquire('./page.summary.controller', {
-    '../../../../presenter/pdf-payload': {pdfPayload: pdfPayloadStub},
-    '../../../../middleware/routes-output/submission-data-with-labels': {submissionDataWithLabels: submissionDataWithLabelsStub},
-    '../../../../page/check-submits/check-submits': {checkSubmits: checkSubmitsStub},
-    '../../../../constants/constants': {
-      SERVICE_OUTPUT_EMAIL: 'bob@gov.uk'
-    }
-  })
-
-  await pageSummaryController.postValidation({}, userDataWithFiles)
-
-  const submissions = submitterClientSpy.getCall(0).args[0].submissions
-  const emailEntries = submissions.filter(s => s.type === 'email')
-  const pdfData = emailEntries[0].attachments[0].pdf_data
-
-  t.deepEquals(pdfData, {some: 'PDF contents'})
-  submitterClientSpy.resetHistory()
-  t.end()
-})
-
-test('USER PDF: it attaches PDF contents to email submission attachments', async t => {
-  getUserDataPropertyStub.resetHistory()
-
-  getUserDataPropertyStub.withArgs('attachUserSubmission').returns(true)
-
-  const pageSummaryController = proxyquire('./page.summary.controller', {
-    '../../../../presenter/pdf-payload': {pdfPayload: pdfPayloadStub},
-    '../../../../middleware/routes-output/submission-data-with-labels': {submissionDataWithLabels: submissionDataWithLabelsStub},
-    '../../../../page/check-submits/check-submits': {checkSubmits: checkSubmitsStub},
-    '../../../../constants/constants': {
-      SERVICE_OUTPUT_EMAIL: 'bob@gov.uk'
-    }
-  })
-
-  await pageSummaryController.postValidation({}, userDataWithFiles)
-
-  const submissions = submitterClientSpy.getCall(0).args[0].submissions
-  const emailEntries = submissions.filter(s => s.type === 'email')
-  const pdfData = emailEntries[0].attachments[0].pdf_data
-
-  t.deepEquals(pdfData, {some: 'PDF contents'})
-  submitterClientSpy.resetHistory()
-  t.end()
-})
```

Comment:
These tests are still relevant.
Covers sending over the PDF data as it currently stands.